app/routes.py

```diff
@@ -1,2 +1,212 @@
-from flask import Blueprint
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime
+import requests
 
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+def customer_response_helper(customer):
+    return {
+            "id": customer.customer_id,
+            "name": customer.name,
+            "registered_at": customer.register_at,
+            "postal_code": customer.postal_code,
+            "phone": customer.phone_number,
+            "videos_checked_out_count": 0
+        }
+
+def video_response_helper(video):
+    return {
+            "id": video.video_id,
+            "title": video.title,
+            "release_date": video.release_date,
+            "total_inventory": video.total_inventory,
+            "available_inventory": video.available_inventory
+        }
+
+def rentals_response_helper(new_rental, customer, video):
+    return {
+            "customer_id": new_rental.customer_id,
+            "video_id": new_rental.video_id,
+            "due_date": new_rental.due_date,
+            "videos_checked_out_count": customer.videos_checked_out_count,
+            "available_inventory": video.available_inventory
+        }
+
+@customers_bp.route("", methods = ["GET"])
+def get_customers():
+    customers = Customer.query.all()
+    customers_response = []
+    if customers is None:
+        return make_response([])
+    for customer in customers:
+        customers_response.append(customer_response_helper(customer))
+    return jsonify(customers_response)
```

Comment:
since `query.all()` returns an empty list when no data found, I'm wondering if this if statement will ever be tripped.  Maybe we could combine these a little more:

```suggestion
def get_customers():
    customers = Customer.query.all()
    customers_response = []
    if len(customers) < 1:
        for customer in customers:
            customers_response.append(customer_response_helper(customer))
    return jsonify(customers_response)
```